### ISSUE TITLE
[A11y]: screen readers read out chat elements when minimized

### ISSUE THREAD

**Issue by tom-youd** [carbon-design-system/carbon-ai-chat#1011]
### Browser

Edge, Chrome

### Operating system

MacOS, Windows

### Package version

1.6.0

### React version (if applicable)

v18

### Automated testing tool and ruleset

n/a

### Assistive technology

JAWS

### Description

Using the JAWS screen reader and with the chat minimized, the screen reader will read out all the elements on the chat which violates WCAG 2.1 Level A requirements.

I've tried added the `aria-hidden` attribute on the parent element (not in the Shadow DOM) and it still reads them out.
I also tried adding the same attribute to all the panels in the Shadow DOM and still reads them out.

I'm happy to provide a fix and will keep trying but does anything have any bright ideas how to solve it?

### Reproduction / example

n/a

### Steps to reproduce

explained above

### Suggested severity

None

### Code of conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-ai-chat/issues) for duplicate problems
